lib/matobj2.gd

```diff
@@ -119,7 +119,25 @@
 ############################################################################
 
 
-# The following are guaranteed to be always set or cheaply calculable:
+#############################################################################
+##
+#A  BaseDomain( <vector> )
+##
+##  <#GAPDoc Label="MatObj_BaseDomain_IsVectorObj">
+##  <ManSection>
+##  <Attr Name="BaseDomain" Arg='vector'/>
+##
+##  <Description>
+##  The semiring <A>vector</A> is defined over.
```

Comment:
Why semiring? Didn't we agree that the `BaseDomain` should be completely unrestricted, i.e. `IsObject`?

(Personally, I am happy with either, just wondering)

Reply:
I'll change it to `IsObject` then. I probably just forgot about that.